### PR TITLE
Fixed double JAR publishing problem with network bootstrapper.

### PR DESCRIPTION
**Pull request by sollecitom**

### CODE REVIEW

File: tools/bootstrapper/build.gradle [hunk 1]
```diff
@@ -5,7 +5,12 @@ apply plugin: 'com.jfrog.artifactory'
 description 'Network bootstrapper'
 
 configurations {
-    runtimeArtifacts
+    runtimeArtifacts.extendsFrom runtime
+    capsuleRuntime
```
Comment (Clintonio) [inline tools/bootstrapper/build.gradle]: Remove this.l

**Comment by josecoll** [comment]
The naming all looks fairly inconsistent now:

corda-tools-blob-inspector (was **tools-blob-inspector**)
archiveName = "blob-inspector

corda-tools-network-bootstrapper (was **tools-network-bootstrapper**)
archiveName "tools-network-bootstrapper (was **network-bootstrapper**)

@tlil  wdyt?

**Comment by tlil** [comment]
Hmm, didn't see this until now. Agree, @josecoll . We should make this more consistent.

**Comment by josecoll** [comment]
So if the fix to this problem was to add

```
jar {
    baseName "corda-tools-network-bootstrapper"
}
```

@sollecitom why did you feel the need to rename the artifacts ?

**Comment by sollecitom** [comment]
As discussed with you and @Clintonio , this seems to be the convention for our published artifacts. Mind that the name of the local JAR (the one that stays unpublished to even the local maven repo, and lives in the `libs` folder) is of little importance.

**Pull request merged**